docs_user/modules/proc_migrating-object-storage-data-to-rhoso-nodes.adoc

```diff
@@ -50,64 +45,64 @@ Sample represents 100.00% of the object partition space
 
 .Procedure
 
-. Add new nodes by scaling up the SwiftStorage resource from 0 to 3. In
-that case 3 storage instances using PVCs are created, running on the
-{rhocp_long} cluster.
+. Add new nodes by scaling up the SwiftStorage resource from 0 to 3:
 // TODO add paragraph / link on EDPM node usage for Swift
 +
 ----
-oc patch openstackcontrolplane openstack --type=merge -p='{"spec":{"swift":{"template":{"swiftStorage":{"replicas": 3}}}}}'
+$ oc patch openstackcontrolplane openstack --type=merge -p='{"spec":{"swift":{"template":{"swiftStorage":{"replicas": 3}}}}}'
 ----
++
+Three storage instances using Persistent Volume Claims are created, running on the {rhocp_long} cluster.
 
 . Wait until all three pods are running:
 +
 ----
-oc wait pods --for condition=Ready -l component=swift-storage
+$ oc wait pods --for condition=Ready -l component=swift-storage
 ----
 
-. Drain the existing nodes. Get the storage management IP
-addresses of the nodes to drain from the current rings:
+. Get the storage management IP addresses of the nodes to drain from the current rings:
```

Comment:
Are the nodes to be drained from the current rings? Or are we retrieving the IP addresses from the current rings?  Or something else? Can we add commas to help clarify?

Reply:
I think we are retrieving the IP addresses from the current rings. @cschwede Can you confirm?

Maybe: "From the current rings, get the storage management IP addresses of the nodes to drain:"

Reply:
Yes, I can confirm - as @klgill  said.